admin/setup_worker_image.sh

```diff
@@ -29,11 +29,15 @@ sudo sh -c 'curl https://storage.googleapis.com/download.dartlang.org/linux/debi
 sudo wget http://master.dl.sourceforge.net/project/d-apt/files/d-apt.list -O /etc/apt/sources.list.d/d-apt.list
 wget -qO - https://dlang.org/d-keyring.gpg | sudo apt-key add -
 
+## Add for Elixir Support
+wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && sudo dpkg -i erlang-solutions_1.0_all.deb
+
 sudo apt-get update
 sudo apt-get -y upgrade
 
 ## List the packages to install for running bots.
-PACKAGES="build-essential gcc g++ python3 python3.6 python3-pip git ocaml openjdk-8-jdk php ruby scala nodejs mono-complete dotnet-dev-1.1.0 libgeos-dev tcl8.5 mit-scheme racket octave luajit lua5.2 ghc erlang-base-hipe coffeescript dart fp-compiler sbcl dmd-bin mono-vbnc gnat-6 cmake python3.6-dev python-numpy cython clang libicu-dev"
+PACKAGES="build-essential gcc g++ python3 python3.6 python3-pip git ocaml openjdk-8-jdk php ruby scala nodejs mono-complete dotnet-dev-1.1.0 libgeos-dev tcl8.5 mit-scheme racket octave luajit lua5.2 ghc erlang-base-hipe coffeescript dart fp-compiler sbcl dmd-bin mono-vbnc gnat-6 cmake python3.6-dev python-numpy cython clang libicu-dev esl-erlang elixir"
```

Comment:
I notice that erlang-base-hipe is already present here @joelbyler , from my limited research, it seems like this could cause issues when installing esl-erlang afterwards?

Reply:
It seems like esl-erlang includes all the packages and erlang-base-hipe could be removed

Reply:
Thanks for checking this out @BobbyMcWho, admittedly I simply just pulled those from the elixir guide but that makes a lot of sense and should clean this up.  Nice catch, thanks!